src/mocking.ts

```diff
@@ -53,6 +55,8 @@ const jestInternalStuff: Array<symbol | string | number> = [
   "@@__IMMUTABLE_RECORD__@@",
   "_isMockFunction",
   "mockClear",
+  "tagName",
+  "hasAttribute",
```

Comment:
Do you know what you did that made Jest access these properties?

Reply:
@eduter This is the minimum code that breaks
```
import originalApi, { Api } from 'game/api'
import { mockInstanceOf } from 'screeps-jest'

jest.mock('game/api')
const api = originalApi as jest.Mocked<Api>
beforeEach(() => {
  jest.clearAllMocks()
})

it('should work', () => {
  const pos = mockInstanceOf<RoomPosition>({ x: 3, y: 3 })
  // other stuff
  expect(api.getOpenSpotsInRange).toHaveBeenCalledWith(pos) // breaks here
})

```
I have no idea why it is trying to print as a DOMElement.
```
 PrettyFormatPluginError: Unexpected access to unmocked property "tagName".
    Did you forget to mock it?
    If you intended for it to be undefined, you can explicitly set it to undefined (recommended) or set "allowUndefinedAccess" argument to true.Error: Unexpected access to unmocked property "tagName".
    Did you forget to mock it?
    If you intended for it to be undefined, you can explicitly set it to undefined (recommended) or set "allowUndefinedAccess" argument to true.

      at Object.get (node_modules/screeps-jest/src/mocking.js:82:23)
      at testNode (node_modules/pretty-format/build/plugins/DOMElement.js:26:31)
      at Object.test (node_modules/pretty-format/build/plugins/DOMElement.js:51:35)
      at findPlugin (node_modules/pretty-format/build/index.js:347:22)
      at prettyFormat (node_modules/pretty-format/build/index.js:527:22)
      at expected.length.expected.map.arg (node_modules/expect/build/spyMatchers.js:35:63)
          at Array.map (<anonymous>)
      at printExpectedArgs (node_modules/expect/build/spyMatchers.js:35:16)
```